### PR TITLE
Move preprocess test and test against "real" data; perform a little cleanup.

### PR DESCRIPTION
**Pull request by lsetiawan**
## Overview

This PR addresses #454. Preprocess test is now moved under preprocess module. There's a `test_old.py` that is now moved under the "old" folder in tests, and extra test_summary_statistics is now deleted.

### CODE REVIEW

**Comment by lsetiawan** [comment]
@leewujung The preprocess mvbs is going to fail in one of the EK80 data. Might need your help to figuring out why. Thanks! 😄

**Comment by leewujung** [comment]
Roger that @lsetiawan. Thanks, I'll take a look!

File: echopype/tests/preprocess/test_preprocess.py [hunk 1]
```diff
@@ -230,3 +281,28 @@ def test_compute_MVBS():
...
+
+
+@pytest.mark.parametrize(param_args, param_testdata)
+def test_preprocess_mvbs(
+    filepath,
+    sonar_model,
+    azfp_xml_path,
+    range_kwargs,
+):
+    # TODO: Need to figure out how to compare the actual rendered plots
```
Comment (leewujung) [inline echopype/tests/preprocess/test_preprocess.py:293]: This line is leftover from another test?

**Comment by leewujung** [comment]
@lsetiawan : the test file `echopype-test-D20211004-T235930.raw` appears to not contain the gain values needed for calibration, causing the test to fail. There is a change better addressed in #477 so that will require pulling those changes here.

**Comment by leewujung** [comment]
Ok, I just fixed that bug and resolved a merge conflict there. Tests passed locally so everything should be fine here once those changes are pulled in there. :)

File: echopype/tests/preprocess/test_preprocess.py (hunk 1 above)
Comment (leewujung) [reply echopype/tests/preprocess/test_preprocess.py]: I removed this and added a docstring.

**Pull request merged**